CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## v1.3.0 (TBA)
 ### General
+* Fix [#278](https://g1cp.org/issues/278): The subtitles of female NPCs now match the audio.
```

Comment:
Would this have to be a bit more specific? To a reader that may be too little information, i.e. which subtitles exactly. Adding the word "SVM" or "Standard Voice Module" may not necessarily aid in any way. Maybe something like "The standard voice subtitles of female NPCs when approaching them now match the audio."